src/iam/certhandler/modules/pkcs11.cpp

```diff
@@ -0,0 +1,997 @@
+/*
```

Comment:
We should move cpp file to separate `pkcs11` folder as well. Similar to `pkcs11.hpp`.

Reply:
fixed.